lib/ci/run_ci.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var FormData = require('form-data');
+
+const {
+  CI_DOMAIN,
+  CI_TYPES,
+  CI_TYPES_KEYS
+} = require('./ci_type_parser');
+
+const CI_CRUMB_URL = `https://${CI_DOMAIN}/crumbIssuer/api/json`;
+const CI_PR_NAME = CI_TYPES.get(CI_TYPES_KEYS.PR).jobName;
+const CI_PR_URL = `https://${CI_DOMAIN}/job/${CI_PR_NAME}/build`;
+
+class RunPRJob {
+  constructor(cli, request, owner, repo, prid) {
+    this.cli = cli;
+    this.request = request;
+    this.owner = owner;
+    this.repo = repo;
+    this.prid = prid;
+  }
+
+  async getCrumb() {
+    try {
+      const { crumb } = await this.request.json(CI_CRUMB_URL);
+      return crumb;
+    } catch (e) {
+      this.cli.error('Failed to get breadcrumbs');
+    }
+  }
+
+  get payload() {
+    const payload = new FormData();
+    payload.append('json', JSON.stringify({
+      parameter: [
+        { name: 'CERTIFY_SAFE', value: 'on' },
+        { name: 'TARGET_GITHUB_ORG', value: this.owner },
+        { name: 'TARGET_REPO_NAME', value: this.repo },
+        { name: 'PR_ID', value: this.prid },
+        { name: 'REBASE_ONTO', value: '<pr base branch>' },
+        { name: 'DESCRIPTION_SETTER_DESCRIPTION', value: '' }
+      ]
+    }));
+    return payload;
+  }
+
+  async start() {
+    this.cli.startSpinner('Validating Jenkins credentials');
+    const crumb = await this.getCrumb();
+
+    if (!crumb) return;
+    this.cli.stopSpinner('Jenkins credentials valid');
+
+    try {
+      this.cli.startSpinner('Starting PR CI job');
+      await this.request.text(CI_PR_URL, {
+        method: 'POST',
+        headers: {
+          'Jenkins-Crumb': crumb
+        },
+        body: this.payload
+      });
+      this.cli.stopSpinner('Starting PR CI job');
```

Comment:
same as below - we can remove the `cli.ok` and say:
```
this.cli.stopSpinner('PR CI job successfully started');
```
(it assumes success by default)